packages/ddp-client/livedata_connection.js

```diff
@@ -33,6 +33,7 @@ var Connection = function (url, options) {
     },
     heartbeatInterval: 17500,
     heartbeatTimeout: 15000,
+    fayeOptions: {},
```

Comment:
Let's call this `npmFayeOptions` to match `npmRequestOptions` in the HTTP package.